autoinstall/Kylin/Server/10/ks.cfg

```diff
@@ -80,8 +80,9 @@ sed -i 's/^#PermitRootLogin .*/PermitRootLogin yes/' /etc/ssh/sshd_config
 # Add new user to sudoer
 echo '{{ new_user }} ALL=(ALL) NOPASSWD:ALL' >/etc/sudoers.d/{{ new_user }}
 # Enable autologin
-sed -i '/\[daemon\]/a AutomaticLogin={{ new_user }}' /etc/gdm/custom.conf
-sed -i '/\[daemon\]/a AutomaticLoginEnable=True' /etc/gdm/custom.conf
+sed -i '/\[Seat:\*\]/a autologin-user={{ new_user }}' /etc/gdm/custom.conf
```

Comment:
The config file path is incorrect. Please also update `Please also update `linux/utils/enable_auto_login.yml `.

Reply:
Thanks. testing is on-going for the changes: check autologin for both deploy new VM and existing VM